keyboards/checkerboards/quark_lp/rules.mk

```
@@ -16,3 +16,4 @@ NKRO_ENABLE = no            # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```

since it looks like the two features share the led strip... probably want to disable rgb light. 
```suggestion
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
```